packages/kbn-alerts-ui-shared/src/alert_filter_controls/alert_filter_controls.tsx

```diff
@@ -0,0 +1,164 @@
+/*
```

Comment:
@XavierM , @umbopepato 

Q. Do we think we could move this to a completely separate package and not couple it with alerts? I don't think filter controls contains any alerts specific code. It takes filters as it is and spits out new set of filters.

Please correct me if I am wrong.

Reply:
I'm not super familiar with the code but I was thinking the same thing!

Reply:
Would it be possible to do in two phases and we will create an issue to move it later on?

Reply:
Also I wanted to added that it theoretically make sense to do it. However, at this point we are the only user of it, so we can wait a little and be practical about it.

Reply:
@XavierM Sure. I am okay with that.